src/node_crypto_common.h

```diff
@@ -13,6 +12,9 @@
 #include <unordered_map>
 
 namespace node {
+
+class Environment;
```

Comment:
Optional nit: You can feel free to leave this out, as `node_crypto.h` is guaranteed to have at least a forward declaration of `Environment`, too.